src/datepicker/datepicker-navigation.ts

```diff
@@ -0,0 +1,56 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+import {NavigationEvent} from './datepicker-view-model';
+import {NgbDate} from './ngb-date';
+import {NgbDatepickerI18n} from './datepicker-i18n';
+import {NgbCalendar} from './ngb-calendar';
+
+@Component({
+  selector: '[ngbDatepickerNavigation]',
+  styles: [`
+    td {
+      text-align: center;
+      padding-bottom: 0.25rem;
+    }
+  `],
+  template: `
+    <tr>
```

Comment:
I know it's restrictive as of now. Not only the `ngbDatepickerNavigation`, but also `ngbDatepickerMonthView` as well. It's because the simplest layout for the calendar I've came up with for now is a table - so all column alignment would be synchronised easily between navigation and dates, weekdays, etc. I would probably migrate to a better one if I knew how to do it easily with purer CSS.

Having said that, the only thing that's exposed now to the user is the template inside of a `<td>` element so the internals should be replaceable in the longer run. Now each section is just a table's `<tbody>`